websites/R/RTLplay/presence.ts

```diff
@@ -0,0 +1,555 @@
+				viewHome: "general.viewHome",
+				viewAccount: "general.viewAccount",
+				viewChannel: "general.viewChannel",
+				viewCategory: "general.viewCategory",
+				viewList: "netflix.viewList",
+				buttonViewPage: "general.buttonViewPage",
+				watching: "general.watching",
+				watchingAd: "youtube.ad",
+				watchingLive: "general.watchingLive",
+				watchingShow: "general.watchingShow",
+				watchingMovie: "general.watchingMovie",
+				listeningMusic: "general.listeningMusic",
+				buttonWatchStream: "general.buttonWatchStream",
+				buttonWatchVideo: "general.buttonWatchVideo",
+				buttonWatchEpisode: "general.buttonViewEpisode",
+				buttonWatchMovie: "general.buttonWatchMovie",
+				buttonListenAlong: "general.buttonListenAlong",
+				live: "general.live",
+				season: "general.season",
+				episode: "general.episode",
+			},
+			await presence.getSetting<string>("lang").catch(() => "en")
+		);
+	};
+let oldLang: string = null,
+	strings: Awaited<ReturnType<typeof getStrings>>;
+
+const enum Assets {
+	Logo = "https://i.imgur.com/KNsI47l.png",
+	Animated = "https://imgur.com/uAqZdFg.gif",
+	Deferred = "https://i.imgur.com/o69hUKD.gif",
+	LiveAnimated = "https://i.imgur.com/6oezfP6.gif",
+	Listening = "https://i.imgur.com/9ZFChOG.png",
+	RTLPlay = "https://i.imgur.com/1f5rMxV.png",
+	RTLTVi = "https://i.imgur.com/wnjbhCe.png",
+	RTLClub = "https://i.imgur.com/8FwZa7m.png",
+	RTLPlug = "https://i.imgur.com/jfYLcJu.png",
+	BelRTL = "https://i.imgur.com/1yZelPm.png",
+	Contact = "https://i.imgur.com/ULP7pgr.png",
+}
+
+function getChannel(channel: string) {
+	switch (true) {
+		case channel.includes("tvi"): {
+			return {
+				channel: "RTL TVi",
+				type: ActivityType.Watching,
+				logo: Assets.RTLTVi,
+			};
+		}
+		case channel.includes("club"): {
+			return {
+				channel: "RTL club",
+				type: ActivityType.Watching,
+				logo: Assets.RTLClub,
+			};
+		}
+		case channel.includes("plug"): {
+			return {
+				channel: "RTL plug",
+				type: ActivityType.Watching,
+				logo: Assets.RTLPlug,
+			};
+		}
+		case channel.includes("bel"): {
+			return {
+				channel: "Bel RTL",
+				type: ActivityType.Listening,
+				logo: Assets.BelRTL,
+			};
+		}
+		case channel.includes("contact"): {
+			return {
+				channel: "Radio Contact",
+				type: ActivityType.Listening,
+				logo: Assets.Contact,
+			};
+		}
+		default: {
+			return {
+				channel,
+				type: ActivityType.Watching,
+				logo: Assets.RTLPlay,
+			};
+		}
+	}
+}
+
+function exist(selector: string) {
+	return document.querySelector(selector) !== null;
+}
+
+const shortenedURLs: Record<string, string> = {};
+async function getShortURL(url: string) {
+	if (!url || url.length < 256) return url;
+	if (shortenedURLs[url]) return shortenedURLs[url];
+	try {
+		const pdURL = await (
+			await fetch(`https://pd.premid.app/create/${url}`)
+		).text();
+		shortenedURLs[url] = pdURL;
+		return pdURL;
+	} catch (err) {
+		presence.error(err);
+		return url;
+	}
+}
+
```

Comment:
```suggestion
let strings: Awaited<ReturnType<typeof getStrings>>

```

Reply:
I had to remove your line in because I already had it in line 41 :)